testhelper/docker/resource/mysql/mysql.go

```diff
@@ -0,0 +1,83 @@
+package mysql
+
+import (
+	"database/sql"
+	_ "encoding/json"
+	"fmt"
+
+	_ "github.com/go-sql-driver/mysql" // mysql driver
+	"github.com/ory/dockertest/v3"
+	dc "github.com/ory/dockertest/v3/docker"
+
+	"github.com/rudderlabs/rudder-go-kit/testhelper/docker/resource"
+)
+
+const (
+	mysqlDefaultDB       = "sources"
+	mysqlDefaultUser     = "root"
+	mysqlDefaultPassword = "password"
```

Comment:
Since we are already in the `mysql` package, do we need `mysql` prefix in here?
```suggestion
	defaultDB       = "sources"
	defaultUser     = "root"
	defaultPassword = "password"
```